samples/echo-bot-aml-deploy/README.md

```diff
@@ -0,0 +1,7 @@
+# Use AMLK SDK to deploy Python Bot to AKS
```

Comment:
AMLK, AKS... What do these acronyms mean to our customers? Why would someone want to deploy to Azure Machine Learning Server and Azure Kubernetes Service?

This README tersely indicates what a user needs to do to get the bot working, but doesn't explain why they would prefer this sample over another, nor what the prerequisites are.